src/app/shared/state/entry-wizard.service.ts

```diff
@@ -0,0 +1,141 @@
+import { Injectable } from '@angular/core';
+import { EntryWizardStore } from './entry-wizard.store';
+import { DefaultService, BioWorkflow } from '../openapi';
+import { AlertService } from '../alert/state/alert.service';
+import { Repository } from '../openapi/model/repository';
+import { HttpErrorResponse } from '@angular/common/http';
+import { finalize } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EntryWizardService {
+  constructor(private entryWizardStore: EntryWizardStore, private defaultService: DefaultService, private alertService: AlertService) {}
```

Comment:
Is DefaultService an intentional name, or was some annotation left out on the webservice side?

Reply:
Probably an annotation left out.

Reply:
Looks like maybe this would solve it
https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Annotations#tag

Reply:
I've made an issue to update the endpoints to have tags.